xonsh/base_shell.py

```diff
@@ -30,6 +30,7 @@ def close(self):
 
     def write(self, data):
         """Writes data to the original stdout and the buffer."""
+        data = data.replace('\001', '').replace('\002', '')
```

Comment:
This was put in because if `$XONSH_STORE_STDOUT is True` in the readline shell, the literal `\001` and `\002` would be printed.  It is probably worth rethinking all of the color management stuff at some point to only insert these characters for the readline shell when needed.  I think that this is the only place they are relevant and we end up removing them in a lot of other places.  Another PR though :)